openwisp_radius/api/serializers.py

```diff
@@ -120,8 +120,12 @@ class RadiusPostAuthSerializer(serializers.ModelSerializer):
         allow_blank=True,
         style={'input_type': 'password'},
     )
-    called_station_id = serializers.CharField(required=False, allow_blank=True)
-    calling_station_id = serializers.CharField(required=False, allow_blank=True)
+    called_station_id = serializers.CharField(
+        required=False, allow_blank=True, max_length=50
+    )
+    calling_station_id = serializers.CharField(
+        required=False, allow_blank=True, max_length=50
+    )
```

Comment:
I would try to remove these two definitions altogether and see if the tests pass anyway.

The reason is that these constraints are already defined in the model and I believe `ModelSerializer` should be smart enough to figure it out without us needing to redefine them again.

Reply:
- Removed the fields `called_station_id` and `calling_station_id` from the `RadiusPostAuthSerializer` and test case still passed.
- Increased the `max_length` of both these fields to `253`.
- Update the test case accordingly.

Will this work?